src/docs/devices/KMP-PRODINo-ESP32EX/index.md

```diff
@@ -190,10 +190,9 @@ binary_sensor:
       inverted: True
 
 light:
-  - platform: fastled_clockless
-    chipset: WS2812B
+  - platform: esp32_rmt_led_strip
+    chipset: ws2812
     pin: GPIO0
     num_leds: 1
-    rgb_order: GRB
```

Comment:
`rgb_order` is required by `esp32_rmt_led_strip`. See the updated issue examples.